running/AutoUpdates/demo.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/bash
+BOLD="$(tput bold)"
+RESET="$(tput sgr0)"
+
+cmd () {
+	echo ""
+	read -p " \$ $1"
+	$1
+}
```

Comment:
This causes the commands to look like they're running rootless even when run as root.....  Not sure if that's good or bad, but it confused me when I first saw it.